src/createServer.js

```diff
@@ -1,3 +1,41 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { validateInputData } = require('./utils/validateInputData');
+const { detectCase } = require('./convertToCase/detectCase');
+
+const PORT = process.env.PORT || 1595;
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, `http://localhost:${PORT}`);
+    const originalText = decodeURIComponent(normalizedUrl.pathname.slice(1));
+    const originalCase = detectCase(originalText);
+    const toCase = normalizedUrl.searchParams.get('toCase');
+
+    res.setHeader('Content-Type', 'application/json');
+
+    const errors = validateInputData(originalText, toCase);
+
+    if (errors.length) {
```

Comment:
Would be better to throw errors in your validate function and catch them here, using `try/catch`